test/data/testvib.py

```diff
@@ -147,17 +147,18 @@ def testatommasses(self):
 class GenericIRimgTest(unittest.TestCase):
     """Generic imaginary vibrational frequency unittest"""
 
+    def setUp(self):
+        self.numvib = 3*len(self.data.atomnos) - 6

```

Comment:
Does this method need to be here if it's the same in the parent class?

Reply:
`GenericIRimgTest` does not inherit from anything other than `unittest.TestCase`. Should this test class be changed to inherit from `GenericIRTest`?